.github/workflows/test.yml

```diff
@@ -23,4 +23,5 @@ jobs:
     - name: Run Unit Tests
       run: |
         npm install
+        npm install file:. --no-save --force
```

Comment:
This allows `cds-types` to import of `@sap/cds`, which then points back to this installation of `cds-types`.  Not needed in out cap/dev setup, but required here in this standalone install.
Otherwise we would always see the release version of `cds-types`.